src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.AcceptAllCerts.cs

```diff
@@ -27,7 +28,7 @@ public void SingletonReturnsTrue()
             Assert.True(HttpClientHandler.DangerousAcceptAnyServerCertificateValidator(null, null, null, SslPolicyErrors.None));
         }
 
-        [Theory]
+        [ConditionalTheory]
```

Comment:
What does it achieve to set this attribute without a condition in it? Is it needed?

Reply:
We use ConditionalFact, ConditionalTheory without conditions when we want to use the SkipTestException() within the test.  It behaves like Fact, Theory, but allows for the SkipTestException() to work.